build_projects/update-dependencies/update-dependencies.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+REPO_ROOT="$DIR/../.."
+PROJECT_PATH="$DIR/update-dependencies.csproj"
+
+# Some things depend on HOME and it may not be set. We should fix those things, but until then, we just patch a value in
```

Comment:
Are you talking about setting `$HOME`?  I was seeing cases where $HOME is unset and that causing issues for `dotnet restore` (since it can't find the home directory to put `.nuget/packages` in during restore).

I'm happy to remove this as all the scinerios I care about have $HOME set, but this could be useful when running the script in Jenkins or maybe Docker.